DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -243,16 +243,16 @@ FROM Centres
             return ipPrefixes ?? new string[0];
         }
 
-        public (bool autoRegistered, string? autoRegisterManagerEmail) GetCentreAutoRegisterValues(int centreId)
-        {
-            return connection.QueryFirstOrDefault<(bool, string?)>(
-                @"SELECT AutoRegistered, AutoRegisterManagerEmail
-                        FROM Centres
-                        WHERE CentreID = @centreId",
-                new { centreId }
-            );
-        }
-
+        public (bool autoRegistered, string? autoRegisterManagerEmail) GetCentreAutoRegisterValues(int centreId)

```

Comment:
Is this correct line endings? @AlexJacksonDS had the same change this morning so one of you must have different setup 🤔

Reply:
Yeah, I think because I used the github merge tool for the original pull request, the line endings got mixed. It's now CRLF.